devices/Chs6540/Chs6540.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Chs6540</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Chs6540</title>
+    <authors>nanoFramework project contributors</authors>
```

Comment:
```suggestion
    <authors>nanoframework</authors>
```

Reply:
shall we adjust for all the bindings? This is the standard one we are using.

Reply:
This has been "fixed" in all libs and several bindings. As it is not critical it's OK to make this adjustment as we touch them.